src/main/java/io/logz/guice/jersey/AbstractContextConfigurator.java

```diff
@@ -0,0 +1,18 @@
+package io.logz.guice.jersey;
+
+import com.google.inject.Injector;
+import org.eclipse.jetty.server.Server;
+
+import java.util.function.Supplier;
+
+class AbstractContextConfigurator {
```

Comment:
I don't see any benefit from having this class when we're making the calls to configure with the implementations.
Please remove this class